pkg/scc/scc.go

```diff
@@ -184,24 +268,94 @@ func (builder *Builder) WithAllowCapabilities(allowCapabilities []coreV1.Capabil
 	return builder
 }
 
+// WithDefaultAddCapabilities adds list of defaultAddCapabilities to SecurityContextConstraints.
+func (builder *Builder) WithDefaultAddCapabilities(defaultAddCapabilities []coreV1.Capability) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"defaultAddCapabilities: %v", builder.Definition.Name, defaultAddCapabilities)
+
```

Comment:
Please add validation step: if len(allowCapabilities) == 0 like in functions above.